src/Features/CSharp/Portable/InitializeParameter/CSharpInitializeMemberFromParameterCodeRefactoringProvider.cs

```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Collections.Immutable;
```

Comment:
remove?

Reply:
It will require some changing to get rid of it. currently it is necessary. I think I can make the changes if you think we should. Don't think we can get rid of 

'''using Microsoft.CodeAnalysis.Operations;''' though